examples/rosa-classic-with-unmanaged-oidc/main.tf

```
@@ -89,15 +90,15 @@ module "rosa_cluster_classic" {
 # machine-pool
 ############################
 module "rosa_machine_pool" {
-  source = "../../modules/machin-pool"
+  source = "../../modules/machine-pool"
 
-  cluster_id = module.rosa_cluster_classic.cluster_id
-  name = var.machine_pool_name
+  cluster_id   = module.rosa_cluster_classic.cluster_id
+  name         = "${var.cluster_name}-MachinePool"
   machine_type = var.machine_type
 
   // Should set one of replicas autoscaling_enabled
-  replicas = var.replicas
+  replicas            = var.replicas
```

I thought we said that here should be a default hardcoded replicas 
```suggestion
  replicas            = 3
```

---

Added the default value to variables